wat

```diff
@@ -1,6 +1,10 @@
 #!/bin/sh
 log_lines() {
-  local regex='^.*\(starting full system upgrade\|upgraded.*\)$'
+  local installed=''
+  if [ "x$SHOW_INSTALLED" == "x1" ]; then
```

Comment:
I don't prefer these style comparisons, and `==` isn't POSIX anyway. Can you use `-eq 1`?